content/en/blog/Kairos-at-FOSDEM-2024.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Kairos at FOSDEM 2024"
+date: 2024-02-06T10:53:13+01:00
+author: Mauro Morales ([Twitter](https://twitter.com/mauromrls)) ([GitHub](https://github.com/mauromorales))
+---
+
+Last year [I said](/blog/2023/02/07/kairos-at-fosdem-2023/) you could expect Kairos to engage further during FOSDEM. And I'm quite pleased to say that's exactly what we did! As part of the open-source ecosystem, we recognize the importance of participating in these types of events. We do so because we understand how critical it to go where our users are, but this year we went a bit further and we also reached out to other similar projects.
+
+## From Free-Lunch to Dog-Fooding: A Culinary Journey in Crafting IaC for Kairos Testing and Building
+
+Dimitris did a fantastic job sharing about how we switched our CI/CD pipelines from using GitHub's free workers to our own based on Kairos. You can check out the slides and recordings [here](https://fosdem.org/2024/schedule/event/fosdem-2024-2983-from-free-lunch-to-dog-fooding-a-culinary-journey-in-crafting-iac-for-kairos-testing-and-building/) (as soon as they become available). We were quite pleased with the level of attendance, and we believe that the message came across.
+
+## Special Purpose Operating Systems Panel (SPOS)
+
+Originally, representatives from Kairos, [Flatcar](https://www.flatcar.org/), and [Unikraft](https://unikraft.org/) wanted to have a panel between the different SPOSs. Unfortunately, FOSDEM didn't accept our request until a the day of the event, so everything had to be done ad-hoc. Which probably ended up being a good thing because we were lucky to also have the participation from [Eve-OS](https://lfedge.org/projects/eve/), [MicroOS](https://microos.opensuse.org/) and [Softiron](https://softiron.com/) on top of the gang which originally signed up.
```

Comment:
```suggestion
Originally, representatives from Kairos, [Flatcar](https://www.flatcar.org/), and [Unikraft](https://unikraft.org/) wanted to have a panel between the different SPOSs. Unfortunately, FOSDEM didn't accept our request until the day of the event, so everything had to be done ad-hoc. Which probably ended up being a good thing because we were lucky to also have the participation from [Eve-OS](https://lfedge.org/projects/eve/), [MicroOS](https://microos.opensuse.org/) and [Softiron](https://softiron.com/) on top of the gang which originally signed up.
```